src/app/on-boarding/on-boarding.component.html

```diff
@@ -27,7 +27,11 @@ <h5 class="card-title fs-14 font-600 br-l ml-2 pl-3">
   <div class="pb-10">
     <button type="button" class="btn btn-primary btn-block btn-lg fs-14 font-weight-bold"
       (click)="register();raiseInteractEvent('register')">{{'REGISTER' | translate}}</button>
-    <button type="button" class="btn btn-outline-primary btn-block fs-14 btn-lg mt-2 font-weight-bold"
+    <button type="button" class="btn btn-primary btn-block btn-lg fs-14 font-weight-bold"
       (click)="login();raiseInteractEvent('login')">{{'LOGIN' | translate}}</button>
+
+     <div>
+      <a class="btn btn-outline-primary btn-block fs-14 btn-lg mt-2 font-weight-bold" href="https://digilocker.meripehchaan.gov.in/public/oauth2/1/authorize?scope=openid&state=We3QtMH9[…]de_challenge_method=S256&nonce=J1ekyldhLyVxCyYXZGmAhg">Login using Meri Pehchaan</a>
```

Comment:
@shreyas1434shinde add translation of  this string

Reply:
@itsvick added